### PR TITLE
Issue#105 gpcp

### PR DESCRIPTION
**Pull request by dnadeau4**
Fix GPCP with wrong coordinate variables longitude(nlon)....

### CODE REVIEW

**Comment by durack1** [comment]
@dnadeau4 this is #105, not #150

File: tests/test_all_formats.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def testHDF(self):
     def testDRS(self):
         f = self.getDataFile("dvtest1.dic")
         data = f['a']
-        self.assertEqual(data.missing_value, 1e20)
+        self.assertEqual(data.time, '15:17:00')
```
Comment (doutriaux1) [inline tests/test_all_formats.py]: why no more missing value?

Comment (dnadeau4) [reply tests/test_all_formats.py]: I asked Ken about that,  do we need a `missing_value` for a DRS file?  It was not in the original file and I added it.  Doing that create `missing_value` for `_bounds` variable which is not `CF-1` compliant.

**Comment by dnadeau4** [comment]
@durack1 you can't be tricked! :smile:

**Comment by dnadeau4** [comment]
@doutriaux1 @durack1 Please check my new solutions.

**Comment by dnadeau4** [comment]
@durack1 @doutriaux1 I do not assign missing_value by default to variables.   This create warning in `cfchecks` especially for lat/lon.   That was done for PP file and @taylor13 mentioned that Ken can set up it's own `missing_value`.

**Comment by durack1** [comment]
@dnadeau4 @doutriaux1 can you trigger a nightly for this PR? It seems the latest nightly is over 1 month old (5th July 2017: https://anaconda.org/uvcdat/cdms2/files)

**Pull request merged**